docs/_layouts/default.html

```diff
@@ -106,5 +122,16 @@ <h1>
         <span id="footerTxt3">{{ label.edit_page[lang] }}</span>
       </a>
     </footer>
+    <script>
+      if ('serviceWorker' in navigator) {
+        window.addEventListener('load', function() {
+          navigator.serviceWorker.register('{{site.baseurl}}/sw.js').then(function(registration) {
+            console.log('ServiceWorker registration successful with scope: ', registration.scope);
+          }, function(err) {
+            console.log('ServiceWorker registration failed: ', err);
+          });
+        });
+      }
+    </script>
```

Comment:
I don't think this is necessary, or even particularly useful. 

Dynamic cache management is useful for "real" web apps (especially if they should work offline as well), but for a simple, static web site like cfw.ftcommunity.de the standard caching mechanism of the browser is more than adequate.

Reply:
This is necessary since this is one of the criteria that must be met for the "Add to Home Screen" banner to be displayed. It also gives the user the ability to access the site without an Internet connection, which is not possible with standard caching. The Progressive Web App score gets increased as well, as it is closely tied to the service worker.

Reply:
OK. I still think it's not very useful, but if Google insists on it they'll get it...